src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,121 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private Long id;
+  private String genre;
```

Comment:
Add the @Colum annotation for the genre attribute

Reply:
Remember to add the `@Column` annotation in genre :)